### ISSUE TITLE
Macos Catalina - Bluetooth not activated 

### ISSUE THREAD

**Issue by chtugha** [50ButtonsEach/hax-with-flic-osx#22]
Hi,

when I klick on the "scan" Button I get this Error:

The scan could not be started since Bluetooth was not activated on this device

but Bluetooth is activated and systemProfiler states: LMP Version: 4.0 (0x6)

**Comment by dmcn42** [comment]
Same issue here. The Flic app works fine but Hax With Flic throws an error.

**Comment by AntonMeier** [comment]
Yes, we have not been maintaining hax-with-flic since the we released the Flic app.

**Comment by atchoo78** [comment]
Here's a workaround mod that I just threw together from bits and pieces, while we wait for Flic 2 Mac app support.
[HaxWithFlic.zip](https://github.com/50ButtonsEach/hax-with-flic-osx/files/4083012/HaxWithFlic.zip)
